packages/@aws-cdk/aws-codebuild/lib/build-spec.ts

```diff
@@ -0,0 +1,101 @@
+import { IResolveContext, Lazy, Stack } from '@aws-cdk/cdk';
+
+/**
+ * BuildSpec for CodeBuild projects
+ */
+export abstract class BuildSpec {
+  public static fromObject(value: {[key: string]: any}): BuildSpec {
+    return new StructuredBuildSpec(value);
+  }
+
+  /**
+   * Use a file from the source as buildspec
+   *
+   * Use this if you want to use a file different from 'buildspec.yml'`
+   */
+  public static fromFilename(filename: string): BuildSpec {
```

Comment:
`fromBuildSource(filePath: string)`

Reply:
I actually like `fromFilename` a lot more, `fromBuildSource` is a weird name in my opinion...